docker/docker-compose.ci.yml

```diff
@@ -48,11 +47,20 @@ services:
     image: unlock
     user: node
     working_dir: /home/unlock/paywall
+    expose:
+      - 3001
+    environment:
+      CI: "${CI}"
+      HTTP_PROVIDER: "${HTTP_PROVIDER}"
+    command: ["npm", "run", "start"]
   integration-tests:
     image: unlock-integration
     user: node
     environment:
       UNLOCK_HOST: "unlock"
+      PAYWALL_URL: "${PAYWALL_URL}"
+      PAYWALL_SCRIPT_URL: "${PAYWALL_SCRIPT_URL}"
```

Comment:
Same as above? Generally since we moved to using the paywall app, could we just rebuild `PAYWALL_SCRIPT_URL` as we need it from the `PAYWALL_HOST` rather than reset it independently every time (more error prone)